.evergreen/config.yml

```diff
@@ -352,6 +352,12 @@ functions:
         permissions: public-read
         content_type: ${content_type|application/gzip}
         display_name: Release Python files all
+  earthly:
+    - command: shell.exec
+      params:
+        shell: bash
+        working_dir: ${working_dir|libmongocrypt}
+        script: bash ${workdir}/libmongocrypt/.evergreen/earthly.sh ${args}
```

Comment:
Are `working_dir` and `workdir` intended to be separate variables?

Reply:
Yes. `workdir` is the directory EVG allocates for the task, but `working_dir` just changes where the subprocess will run. For now, it isn't important to tweak it, since there's only one directory with an Earthfile in it. The similarity in the variable names is unfortunate, I suppose.